www/src/app/pages/dashboard-page/dashboard-page.component.ts

```diff
@@ -1,26 +0,0 @@
-import {Component, OnInit} from '@angular/core';
```

Comment:
Same question as above

Reply:
Yes, this is the old Dashboard component for instructors. They are now using the admin grid as their dashboard.